app/models/watched_word_group.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class WatchedWordGroup < ActiveRecord::Base
+  validates :action, presence: true
+
+  has_many :watched_words, dependent: :destroy
+
```

Comment:
We should also probably reference the same actions enum as `WatchedWord`:

https://github.com/discourse/discourse/blob/3eecc293542d65d0683754be447cab7a22faab88/app/models/watched_word.rb#L37-L50

So doing this might work:

```ruby
def self.actions
  WatchedWord.actions
end
```